gitlab/README.md

```diff
@@ -22,16 +22,14 @@ Follow the instructions below to configure this check for an Agent running on a
 
 ##### Metric collection
 
-1. Edit the `gitlab.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][3], to point to the Gitlab's Prometheus metrics endpoint. See the [sample gitlab.d/conf.yaml][4] for all available configuration options.
+1. Edit the `gitlab.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][3], to point to the Gitlab's metrics endpoint. See the [sample gitlab.d/conf.yaml][4] for all available configuration options.
```

Comment:
```suggestion
1. Edit the `gitlab.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][3], to point to the Gitlab's metrics [endpoint](https://docs.gitlab.com/ee/administration/monitoring/prometheus/gitlab_metrics.html#collecting-the-metrics). See the [sample gitlab.d/conf.yaml][4] for all available configuration options.
```
Let's also add a link to the Gitlab documentation for more info.

Reply:
Not sure if it's clear for gitlab users, but we should also document that the IP needs to be whitelisted in order to monitor gitlab https://docs.gitlab.com/ee/administration/monitoring/ip_whitelist.html

Reply:
True, that's something that should be included

Reply:
Added!